cmd/web.go

```diff
@@ -62,7 +62,7 @@ and it takes care of all the other things for you`,
 		},
 		cli.StringFlag{
 			Name:  "pid, P",
-			Value: "custom/run/app.pid",
+			Value: "/var/run/gitea.pid",
```

Comment:
`/var/run/` [Is the normal location for pid files](http://stackoverflow.com/a/5174433)